commands/General/Role Manipulation/roleme.js

```diff
@@ -0,0 +1,86 @@
+const { Command } = require('klasa');
+const { RichDisplay } = require('klasa');
+const { MessageEmbed } = require('discord.js');
+
+/* You need to add the following lines to your guild schematic manager:
+     	.add('selfroles', folder => folder
+		.add('roles', 'role', { array: true })
+            	.add('enabled', 'boolean', { default: true }))
+*/
+
+module.exports = class extends Command {
```

Comment:
This command looks like a carbon copy of manageroles.js up above... And from the file names, I assume that's unintended.